models/sweeps/define-sweep-configuration.mdx

```diff
@@ -74,52 +74,197 @@ Within the top level `parameters` key, the following keys are nested: `learning_
 
 ## Double nested parameters
 
-Sweep configurations support nested parameters. To delineate a nested parameter, use an additional `parameters` key under the top level parameter name. Sweep configs support multi-level nesting.
+Sweep configurations support nested parameters. To define a nested parameter, include an additional `parameters` key under the top-level parameter name.
```

Comment:
In the code snippets, it looks like `parameters` is a top-level key. Maybe that's my misunderstanding.